packages/sdk/src/Campaigns.ts

```diff
@@ -39,14 +39,14 @@ export class Campaigns {
 	 * @see https://docs.voucherify.io/reference/delete-campaign
 	 */
 	public delete(name: string, params: T.CampaignsDeleteParams = {}) {
-		return this.client.delete(`/campaigns/${encode(name)}`, params)
+		return this.client.delete<T.CampaignsDeleteCampaignResponse>(`/campaigns/${encode(name)}`, params)
 	}
 	/**
 	 * @see https://docs.voucherify.io/reference/add-voucher-to-campaign
 	 */
 
-	public addVoucher(name: string, body: T.CampaignsAddVoucherBody = {}, params: T.CampaignsAddVoucherParams = {}) {
```

Comment:
It's a breaking change and on top of that, such call:
`voucherify.campaigns.addVoucher('test-02')` is completely valid as both body and params are not required